test/test_cases/TEST_ID_2_multiple_pods_changing_pwd_inbetween.sh

```diff
@@ -14,19 +14,38 @@ pod_name1=$(cli_get_pods_test_env | awk '{print $1}')
 echo "Verify pod $pod_name1 has environment variable 'TEST_SECRET' with value 'supersecret'"
 verify_secret_value_in_pod $pod_name1 TEST_SECRET supersecret
 
-test_app_set_secret secrets/test_secret secret2
-
 echo "Deleting pod $pod_name1"
-$cli delete pod $pod_name1
+if [[ "$PLATFORM" = "kubernetes" ]]; then
+    # Using scaling down instead of delete because in GKE, deleting a pod automatically triggers the spin up of another one
+    # and therefore two pods were begin returned when parsing the namespace which messes up the test. While with scale down,
+    # we can are able to successfully delete the pods and only deploy a new one, when we scale back up.
+    $cli scale deployment test-env --replicas=0
+
+    # Waiting until pod is successfully removed from the namespace before advancing.
+    wait_for_it 600 "$cli get pods --namespace=$TEST_APP_NAMESPACE_NAME --selector app=test-env --no-headers | wc -l | tr -d ' ' | grep '^0$'"
+
+    test_app_set_secret secrets/test_secret secret2
+
+    $cli scale deployment test-env --replicas=1
+elif [ $PLATFORM = 'openshift' ]; then
```

Comment:
please align if we use double-quotes or quotes.

Reply:
done